frameworks/cassandra/src/test/java/com/mesosphere/sdk/cassandra/scheduler/CassandraRecoveryPlanOverriderTest.java

```diff
@@ -11,64 +11,45 @@
 import com.mesosphere.sdk.state.StateStore;
 import com.mesosphere.sdk.state.StateStoreUtils;
 import com.mesosphere.sdk.storage.MemPersister;
-import com.mesosphere.sdk.testing.BaseServiceSpecTest;
+import com.mesosphere.sdk.testing.ServiceRenderUtils;
 import org.apache.mesos.Protos;
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Test;
 
+import java.io.File;
 import java.util.Arrays;
+import java.util.Collections;
+import java.util.Map;
 import java.util.Optional;
 import java.util.UUID;
 
 /**
  * This class tests Cassandra's custom replacement of nodes.
  */
-public class CassandraRecoveryPlanOverriderTest extends BaseServiceSpecTest {
+public class CassandraRecoveryPlanOverriderTest {
```

Comment:
If we kept a base class, it seems like we could lose all of the constructor to a super, no?

Reply:
Ah, I see that the ServiceSpecTests all are very simple.

I just worry about having this much setup for other tests.

Reply:
Redid the testing API, now there's much less boilerplate here.

Reply:
👍